Ginger/Ginger/Drivers/DriversWindows/MobileDriverWindow.xaml.cs

```diff
@@ -148,10 +245,40 @@ private async void MobileDriverWindow_DriverMessageEvent(object sender, DriverMe
                             xMessagePnl.Visibility = Visibility.Collapsed;
                             xDeviceScreenshotCanvas.Visibility = Visibility.Visible;
                             xMessageLbl.Content = "Loading Device Screenshot...";
+
+                            //Loading image on details and metrics ucGrid
```

Comment:
on device load no need to details with details/metrics controls or data besides getting the needed data for title of the window, rest needs to be done only when used clicks on the "Metrics" icon on tool bar